grass7/raster/r.sim.terrain/r.sim.terrain.html

```diff
@@ -93,19 +93,25 @@ <h2>EXAMPLES</h2>
 <h2>ERROR MESSAGES</h2>
 
 If the module fails with
-
 <div class="code"><pre>
 ERROR: nwalk (7000001) &gt; maxw (7000000)!
 </pre></div>
-
```

Comment:
This error should not show up anymore with 7.8.5 I believe, the restriction on number of cells is removed. The advice would not work anyway. I suggest removing this from here.